### PR TITLE
Adding setExpiryMilliseconds function

### LINKED ISSUE #71: Expiry time less or a fraction of a minute
Does the expiry time have to be a whole minute and it can it be less than a minute?

### PR DESCRIPTION
**Pull request by mikehedman**
Fixes #71 

I altered one test that was displaying somewhat deceiving results.  The line:
`lscache.set(expiredKey, 'blech', 1/60);`
actually caused that item to immediately expire.  The first get() call always returns null, no matter how soon you call it.  So I reworked that test using my new function.

### CODE REVIEW

File: lscache.js [hunk 1]
```diff
@@ -355,6 +358,27 @@
       cacheBucket = '';
     },
 
+    /**
+     * @returns {number} The currently set number of milliseconds each time unit represents in the set() function's "time" argument.
+     */
+    getExpiryMilliseconds: function() {
```
[the remark this answers is not in the document]
Comment (mikehedman) [reply lscache.js]: I can't think of a reason, maybe if someone wanted to have "responsive" durations that evolve based on some situation.  But yeah, I added it for use in the tests, and to be symmetrical with the setter.  I don't think it hurts anything, and it does make it easier to test, so for me, I vote to keep it.

File: lscache.js [hunk 2]
```diff
@@ -355,6 +358,27 @@
       cacheBucket = '';
     },
 
+    /**
+     * @returns {number} The currently set number of milliseconds each time unit represents in the set() function's "time" argument.
```
[the remark this answers is not in the document]
Comment (mikehedman) [reply lscache.js]: Done

[the remark this answers is not in the document]
Comment (mikehedman) [reply lscache.js]: There are a couple of other comments, and one line of code that goes over the 100 char limit.  I didn't want to start messing around outside of my scope, so I left them.

File: lscache.js [hunk 3]
```diff
@@ -355,6 +358,27 @@
...
 
+    /**
+     * @returns {number} The currently set number of milliseconds each time unit represents in the set() function's "time" argument.
+     */
+    getExpiryMilliseconds: function() {
+      return expiryMilliseconds;
+    },
+
+    /**
+     * Sets the number of milliseconds each time unit represents in the set() function's "time" argument.
```
[the remark this answers is not in the document]
Comment (mikehedman) [reply lscache.js]: Done

File: lscache.js [hunk 4]
```diff
@@ -355,6 +358,27 @@
...
+     * Sample values:
+     *  1: each time unit = 1 millisecond
+     *  1000: each time unit = 1 second
+     *  60000: each time unit = 1 minute (Default value)
+     *  360000: each time unit = 1 hour
+     * @param {int} milliseconds
```
[the remark this answers is not in the document]
Comment (mikehedman) [reply lscache.js]: Done

**Comment by mikehedman** [comment]
Added commit with requested changes.  Only includes 2 files, since the changes were just comments, so they are not included in the .min file (took me a second to realize that).

**Comment by pamelafox** [comment]
Thanks for the changes, I've merged and will tag a new release shortly.

**Pull request merged**